match-db/init.sql

```diff
@@ -1,6 +1,7 @@
 CREATE TABLE match (
-    id SERIAL PRIMARY KEY,
-    userOne int,
-    userTwo int,
-    matchedOn timestamp
-  );
+  id SERIAL PRIMARY KEY,
+  auth_id INT NOT NULL,
+  userOne INT,
+  userTwo INT,
+  matchedOn TIMESTAMP,
```

Comment:
Think we've been pretty consistent with the 4 space indentation in these files

Reply:
Fair, only did it because the Postgres creation scripts we generate indent with 2 spaces